src/metabase/api/database.clj

```diff
@@ -249,42 +256,31 @@
 
   * `saved` means we should include the saved questions virtual database. Default: `false`.
 
-  * `include_tables` is a legacy alias for `include=tables`, but should be considered deprecated as of 0.35.0, and will
-    be removed in a future release.
-
-  * `include_cards` here means we should also include virtual Table entries for saved Questions, e.g. so we can easily
-    use them as source Tables in queries. This is a deprecated alias for `saved=true` + `include=tables` (for the saved
-    questions virtual DB). Prefer using `include` and `saved` instead.
-
   * `include_editable_data_model` will only include DBs for which the current user has data model editing
     permissions. (If `include=tables`, this also applies to the list of tables in each DB). Should only be used if
     Enterprise Edition code is available the advanced-permissions feature is enabled.
 
   * `exclude_uneditable_details` will only include DBs for which the current user can edit the DB details. Has no
-    effect unless Enterprise Edition code is available and the advanced-permissions feature is enabled."
-  [include_tables include_cards include saved include_editable_data_model exclude_uneditable_details]
-  {include_tables                [:maybe :boolean]
-   include_cards                 [:maybe :boolean]
-   include                       (mu/with-api-error-message
+    effect unless Enterprise Edition code is available and the advanced-permissions feature is enabled.
+
+  * `include_only_uploadable` will only include DBs into which Metabase can insert new data."
```

Comment:
how about providing `uploadable` as an option for `include` param? looks like we're wanting to go toward using one single param with multiple values instead of multiple boolean params

Reply:
ofc that means we want `include` to be either a single string or a seq.

see the `exclude_ids` for `GET /:id/series` as an example of such hybrid option https://github.com/metabase/metabase/blob/0a43e4ab1d5d92ece50b241f88bc6fb083c51c76/src/metabase/api/card.clj#L360

Reply:
I'm not sure I like mixing the semantics of `include` handling both hydration (of tables) and filtering (of databases).